pkg/cli/util/args.go

```diff
@@ -67,3 +69,24 @@ func NamedPositionalArgsValidator(failMissing, failExtra bool, expectedArgs ...s
 		return nil
 	}
 }
+
+// PromptForMissingArgs expects that the number of args, matched the number of argNames, in the order they should appear
+// and will prompt one by one for the missing args adding them to the args slice and returning a new set for a command to use
+func PromptForMissingArgs(l log.Logger, args []string, argNames ...string) ([]string, error) {
+	if len(args) == len(argNames) {
+		return args, nil
+	}
+
+	for i := range argNames[len(args):] {
```

Comment:
We should handle the case where this is not a interactive terminal as in a pipeline you would never know why this command has not succeeded as it just hangs for the question